includes/class-donations.php

```diff
@@ -776,6 +776,18 @@ function ( $item ) {
 			$is_wc ? \wc_get_page_permalink( 'checkout' ) : ''
 		);
 
+		$donation_metadata = [
+			'currency'   => \get_woocommerce_currency(),
+			'amount'     => $donation_value,
+			'product_id' => $product_id,
+			'referer'    => $referer,
+		];
+
+		/**
+		 * Action to fire for donate checkout modal.
+		 */
+		\do_action( 'newspack_blocks_donate_block_modal', $donation_metadata );
```

Comment:
where is this action used?

Reply:
Ah good catch, @leogermani! 🤦‍♀️ It doesn't look like it is anymore -- it was cruft left over from when more of the events were handled in the backend. It's been removed in [9c781a2](https://github.com/Automattic/newspack-plugin/pull/3297/commits/9c781a2398e4ffeb7faa3aa7b7d4976058780be1)!